spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe PostsController do
+
+  let(:valid_attributes) {{title: 'Lorem', body: 'Ipsum'}}
```

Comment:
Space missing inside {.<br>Space missing inside }.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.